superset/utils.py

```diff
@@ -513,6 +513,17 @@ def table_has_constraint(table, name, db):
     return False
 
 
+def upload_file(file):
+    """Takes in the name of the file to be uploaded and uploads that file"""
+
+    from superset import app
+    config = app.config
+    if not file or not file.filename:
```

Comment:
What happens when this `Exception` gets raised? Note in the previous version this check doesn't exist.

Reply:
It gets caught in the try block in views/core.py and the error message gets shown at the top of the form

Reply:
Can this happen in practice? You have one caller of this function so you can be pretty sure that a proper file is passed down.

Reply:
True. I'll take it out.